src/ui/server/helpers/mappings/map-countries.ts

```diff
@@ -82,6 +82,7 @@ export const filterCisCountries = (countries: Array<CisCountry>) => countries.fi
 export const mapCountry = (country: CisCountry, selectedIsoCode?: string): Country => {
   const mapped = {
     name: country.marketName,
+    text: country.marketName,
```

Comment:
can we use name instead where 'text' is being referred to? as it seems like duplication otherwise? Ignore me if I am misunderstanding

Reply:
We need to revert these changes to `map-countries` and the country type change (in `types/country.d.ts`), this is my fault for not making this clear. 

`map-countries` was written a while ago and is used for mapping countries we get from an external API. it's not used to map data to the structure we need for the GOV select component, like `map-currencies`.

I will create a seperate PR to rename `map-countries` so this is clearer, then we can create a new `map-countries` like `map-currencies`. This is required to pass the A/C of this ticket - without a new mapping, the validation won't work because something will always be selected.